src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -1649,6 +1649,86 @@ public static String convertHG19ContigToB37Contig( final String hg19Contig ) {
         return HG19_TO_B37_CONTIG_NAME_MAP.getOrDefault(hg19Contig, hg19Contig);
     }
 
+    /**
+     * Create a cDNA string for an intronic variant.
+     * The cDNA string contains information about the coding position of a variant and the alleles involved.
+     * If the transcript in which the variant occurs contains at least 1 exon, the string will be non-empty and of
+     * the form:
+     *     c.e[EXON NUMBER][+|-][BASES FROM EXON][REF ALLELE]>[ALT ALLELE]
+     * Concretely:
+     *     c.e2-1A>G
+     * Where:
+     *      2 = the number of the exon to which the given variant start is closest
+     *     -1 = number of bases away from the exon (1 before)
+     *      A = Reference allele
+     *      G = Alternate allele
+     * @param variantStart The start position (1-based, inclusive) in genomic coordinates of the variant.
+     * @param exonList The {@link List<Locatable>} representing the exons in the transcript in which this variant occurs.
+     * @param strandCorrectedRefAllele A {@link String} containing the bases of the reference allele, which are correct for strandedness (i.e. if on a transcript on the {@link Strand#NEGATIVE} strand, the string has already been reverse-complemented).
+     * @param strandCorrectedAltAllele A {@link String} containing the bases of the alternate allele, which are correct for strandedness (i.e. if on a transcript on the {@link Strand#NEGATIVE} strand, the string has already been reverse-complemented).
+     * @return A {@link String} representing the cDNA change for the given data.  Will be empty if the given {@code exonList} is empty.
+     */
+    public static String createIntronicCDnaString(final int variantStart,
```

Comment:
Can  all of these parameters be `null`?  If not, please add a comment in the parameter javadoc (`Never {@code null}` and check the values with `Utils.nonNull(...)`  ...  This should be done for all public methods.

Reply:
It was an oversight leaving it out.

Fixed!